spi/identity-hub-store-spi/src/testFixtures/java/org/eclipse/dataspaceconnector/identityhub/store/spi/IdentityHubStoreTestBase.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2020 - 2022 Amadeus
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Amadeus - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.identityhub.store.spi;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.dataspaceconnector.identityhub.credentials.model.VerifiableCredential;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public abstract class IdentityHubStoreTestBase {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+    private static final Random RAND = new Random();
+
+    @Test
+    void saveAndListVerifiableCredentials() {
```

Comment:
The test coverage could be extended, eg.: 
- getAll_whenNothingInDb
- add_whenExists